scripts/start-headless.sh

```
@@ -0,0 +1,29 @@
+
+# Cleanup existing instances if any
+pkill -f "Xvfb.*:99" || true
+pkill -f "fluxbox.*:99" || true
+rm -f /tmp/.X99-lock /tmp/.X11-unix/X99
+
+# Start Xvfb
+Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+echo "Xvfb started on :99"
+
+# Wait for Xvfb to be ready
+for i in {1..50}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)